service-commands/package.json

```diff
@@ -12,15 +12,19 @@
     "colors": "^1.4.0",
     "commander": "^6.0.0",
     "convict": "^5.2.0",
-    "dotenv": "^8.2.0",
     "ffmpeg-static": "^4.2.7",
     "node-fetch": "^2.6.0",
     "untildify": "^4.0.0",
     "web3": "^1.2.7",
     "@solana/web3.js": "1.20.0",
-    "borsh": "^0.4.0"
+    "borsh": "^0.4.0",
```

Comment:
Added some dependencies here to support the logger (duplicated from mad-dog; chatted w @vicky-g ) and random util fns (moved from mad-dog).